experimental/examples/imageviewer/shared/src/desktopMain/kotlin/example/imageviewer/Localization.desktop.kt

```diff
@@ -0,0 +1,8 @@
+package example.imageviewer
+
+actual fun getCurrentLanguage(): AvailableLanguages =
+    if (System.getProperty("user.language").equals("de")) {
```

Comment:
Let's convert it to "when" to minimize changes in case of adding new language

Reply:
Done